drivers/scheduler/k8s/k8s.go

```diff
@@ -429,10 +439,18 @@ func (k *k8s) createNamespace(app *spec.AppSpec, namespace string) (*v1.Namespac
 	return nsObj.(*v1.Namespace), nil
 }
 
-func (k *k8s) createStorageObject(spec interface{}, ns *v1.Namespace, app *spec.AppSpec) (interface{}, error) {
+func (k *k8s) createStorageObject(spec interface{}, ns *v1.Namespace, app *spec.AppSpec, storageprovisioner string) (interface{}, error) {
 	k8sOps := k8s_ops.Instance()
+
 	if obj, ok := spec.(*storage_api.StorageClass); ok {
+		logrus.Infof("[%v] Requested provisioner: %s", app.Key, storageprovisioner)
```

Comment:
print this only if it is non-empty. Else this will add an extra log line for default jobs too.

Reply:
Done.